### PR TITLE
One for the Pyne Maintainer in his dark office

### PR DESCRIPTION
**Pull request by bam241**
_One python script to build them all,
One python script to configure them,
One python script to name them,
and on the cloud upload them,
In DockerHub where the docker images live._

### CODE REVIEW

File: docker/make_pyne_docker_image.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import shutil
+import io
+import argparse as ap
+
+
+def absexpanduser(x): return os.path.abspath(os.path.expanduser(x))
+
+
+def consistancy_check(args):
+    args.moab = args.moab or args.dagmc or args.pymoab or args.all
```
Comment (kkiesling) [inline docker/make_pyne_docker_image.py:17]: doc string?

**Review by kkiesling — COMMENTED** [review]
just doc strings (per request of @bam241 ) and then it looks fine to me.

**Comment by bam241** [comment]
@kkiesling should be good to go :)

**Comment by kkiesling** [comment]
thanks @bam241

**Pull request merged**